### DISCUSSION TITLE
How I replace the post.py `_set_date` behavior

### DISCUSSION THREAD

**Discussion by huhuang03** [getnikola/nikola#3522] [Q&A]
My posts don't have a date meta data. It will raise `ValueError: Missing date in file posts/README.org` when scan posts.

What I want is change this behavior, some thing like just skip scan this post.

If I don't want modify the source code directly, can I do this by write a plugin? And how?

**Comment by ralsina** [comment]
Posts need dates because they are usually shown ordered by date.

If you want things that are not ordered by date, use pages instead of posts.
